src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,34 @@
+package calculator;
+
+import java.util.Objects;
+
+public class StringCalculator {
+    public int stringCalculate(String input) {
+        if(Objects.isNull(input) || input.isEmpty()) {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        if (Objects.isNull(input) || input.isEmpty()) {
```